odd_collector_gcp/adapters/bigtable/adapter.py

```diff
@@ -0,0 +1,56 @@
+from typing import List
+
+from odd_collector_sdk.domain.adapter import AbstractAdapter
+from odd_models.models import DataEntityList
+from google.cloud import bigtable
+from oddrn_generator import BigTableGenerator
+
+from odd_collector_gcp.adapters.bigtable.mapper import BigTableMapper
+from odd_collector_gcp.adapters.bigtable.models import (
+    BigTableInstance,
+    BigTableTable,
+    BigTableColumn,
+)
+from odd_collector_gcp.domain.plugin import BigTablePlugin
+
+
+class Adapter(AbstractAdapter):
+    def __init__(self, config: BigTablePlugin):
+        self.__client = bigtable.Client(project=config.project, admin=True)
+        self.__generator = BigTableGenerator(
+            google_cloud_settings={"project": config.project}
+        )
+        self.__mapper = BigTableMapper(oddrn_generator=self.__generator)
+
+    def get_data_source_oddrn(self) -> str:
+        return self.__generator.get_data_source_oddrn()
+
+    def get_data_entity_list(self) -> DataEntityList:
+        entities = self.__mapper.map_instances(self.__get_instances())
+
+        return DataEntityList(
+            data_source_oddrn=self.get_data_source_oddrn(), items=entities
+        )
+
+    def __get_instances(self) -> List[BigTableInstance]:
+        instances = []
+        for instance in self.__client.list_instances()[0]:
```

Comment:
What is `self.__client.list_instances()[0]` ? Please explicit what is the first element.